### ISSUE TITLE
No support for numpy types

### ISSUE THREAD

**Issue by AustinJRoberts** [dwavesystems/dwave-inspector#127]
It seems that if numpy types are used in a bqm, json can't serialize it and inspector does not work.

Code below gives error in browser window that opens.
```
import numpy as np
import dimod
from dwave.system import DWaveSampler, FixedEmbeddingComposite

import dwave.inspector 

a = np.int32(5)
bqm = dimod.BinaryQuadraticModel({a:a},{}, 'BINARY')

qpu = DWaveSampler()
sampler = FixedEmbeddingComposite(qpu, {a:[105]})
sample_set = sampler.sample(bqm)

dwave.inspector.show(bqm, sample_set)
```

**Comment by randomir** [comment]
Thanks @AustinJRoberts! I can reproduce the issue. It should be an easy fix.

**Comment by randomir** [comment]
Might be a duplicate of #106.

**Comment by randomir** [comment]
This will be fixed with #81.

**Comment by randomir** [comment]
Important to note: JSON serialization of model variables/biases, answer and metadata has already been fixed in #47. What's different here are NumPy variables in the embedding. If `FixedEmbeddingComposite` is omitted, the code from above works.

```
import numpy as np
import dimod
from dwave.system import DWaveSampler, FixedEmbeddingComposite

import dwave.inspector 

a = np.int32(30)
bqm = dimod.BinaryQuadraticModel({a:a},{}, 'BINARY')

sampler = DWaveSampler()
sample_set = sampler.sample(bqm)

dwave.inspector.show(bqm, sample_set)
```
